tests/benchmark/run-benchmarks.sh

```diff
@@ -60,4 +60,4 @@ sleep 2
     --address "${CLIENT_SOCK}" \
     --runtime.id "8000000000000000000000000000000000000000000000000000000000000000" \
     --benchmarks accounts_transfers \
-    --benchmarks.concurrency 4000
+    --benchmarks.concurrency 3000
```

Comment:
Not related to changes here, but hopefully this reduces the flakiness of the benchmarks CI job (works fine locally).